Bio/PDB/Atom.py

```diff
@@ -262,6 +263,40 @@ def __sub__(self, other):
         diff = self.coord - other.coord
         return np.sqrt(np.dot(diff, diff))
 
+    def strictly_equals(self, other: "Atom") -> bool:
+        """Compare this atom to the other atom using a strict definition of equality.
+
+        Return true if the atoms have the same name, coordinates, B factor, occupancy,
+        alternate location indicator (altloc), and fullname.
+        Return false otherwise.
+
+        If these atoms are equal, assert that the element symbol, atomic charge, and
+        radius are equal if these attributes are present.
+
+        :param other: The atom to compare this atom with
+        :type other: Atom
+        :return: Whether the atoms are strictly equal
+        :rtype: bool
+        """
+        if not isinstance(other, type(self)):
+            return False
+
+        are_equal = (
+            self.name == other.name
+            and np.allclose(self.coord, other.coord)
+            and self.bfactor == other.bfactor
+            and self.occupancy == other.occupancy
+            and self.altloc == other.altloc
+            and self.fullname == other.fullname
+        )
+
+        if are_equal:
```

Comment:
Why not add these to the `are_equal` above with:

```python
getattr(self, "element", None) == getattr(self, "element", None)
...
```

We'd have to update this function in any case if the attributes change so doesn't seem a lot more work to have the checks explicit there.